src/renderer/shared/components/Confirm/Confirm.js

```diff
@@ -57,10 +58,17 @@ export default class Confirm extends React.PureComponent {
             {children}
           </div>
           <div className={styles.actions}>
-            <Button className={styles.action} ref={this.registerRef('confirm')} onClick={onConfirm}>
+            <PrimaryButton
+              className={styles.action}
+              ref={this.registerRef('confirm')}
+              onClick={onConfirm}
+            >
               {confirmLabel}
-            </Button>
-            <Button className={classNames(styles.action, styles.cancel)} onClick={onCancel}>
+            </PrimaryButton>
+            <Button
```

Comment:
maybe rename SecondaryButton? It's not very clear which style is applied from the name

Reply:
There are places where we only display a single button and it's not secondary to anything, which is why I left it as just `Button`.  It's only a secondary button, when there's not a primary button visible, but otherwise it's just the default button style.